apps/thismypc/thismypc.yml

```
@@ -0,0 +1,12 @@
+name: ThisMyPC
+description: ' It is a full stack based project that starts as an experiment.Thismypc is software that you can use to browse your pc remotely from any browser

```

Looks like CI is failing because the description is missing an endquote, so this suggestion adds that and shortens the description to a one-sentence explanation of what the software is used for.

Feel free to rephrase if this is _too_ short, but please keep it in the spirit of other app descriptions.

```suggestion
description: 'Browse your PC remotely from any web browser.'
```

---

@ckerr  Ok thanks